### ISSUE TITLE
✨ [Feature] Support withSum, withAvg, withMax, withMin, withCount methods

### ISSUE THREAD

**Issue by liushuai05** [goravel/goravel#501]
### Before feedback (在反馈之前)

- [X] There are no features that I want to submit in Issues (当前 Issues 中没有我将要提交的新功能)

### Describe Feature (描述功能)

希望能支持类似larvel的 withSum ， withAvg, withMax,withMin，withCount等方法

**Comment by hwbrzzl** [comment]
Good idea, we need to make an investigation first, to check if they can be supported in framework.
